cognite/client/_api/data_modeling/instances.py

```diff
@@ -1093,10 +1093,11 @@ def _query_or_sync(self, query: Query, endpoint: Literal["query", "sync"]) -> Qu
     @overload
     def list(
         self,
+        *,
         instance_type: Literal["node"] = "node",
-        include_typing: bool = False,
+        include_typing: Literal[False] = False,
```

Comment:
Let's add a comment about why for this.

Reply:
That was a mistake. I did not intend to do this change yet, that is coming in the next PR.